src/MangaStream.ts

```diff
@@ -141,6 +141,11 @@ export abstract class MangaStream implements ChapterProviding, HomePageSectionsP
      */
     directoryPath = 'manga'
 
+    /**
+     * The pathname between the domain and the filter page (this is usually the same as the directory path).
+     */
+    filterPath = this.directoryPath
```

Comment:
What's the purpose for this and why should it be added? Has not been needed before?

Reply:
I'm trying to seperate values to make it future-proof. In hindsight, I don't think it'll be useful anymore and will remove it in a subsequent patch commit.